lib/Service/ProvisioningService.php

```diff
@@ -398,6 +398,11 @@ public function getSyncGroupsOfToken(int $providerId, object $idTokenPayload) {
 		if ($event->hasValue() && $event->getValue() !== null) {
 			// casted to null if empty value
 			$groups = json_decode($event->getValue() ?? '');
+			// support values like group1,group2
+			if (is_string($groups)) {
+				$groups = explode(',', $groups);
+				$groups = array_filter($groups);
```

Comment:
In case there is any excess whitespace in the string, maybe it would also be a good idea to use `trim`/`mb_trim` to remove them.
wdyt?